.travis.yml

```diff
@@ -0,0 +1,35 @@
+sudo: false
+language: go
+go:
+- 1.5.4
+- 1.6.2
+env:
+  global:
+    - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/intelsdi-x/snap-plugin-publisher-heka
+    - HEKA_SOURCE=/home/travis/gopath/src/github.com/mozilla-services/heka
+  matrix:
+    - TEST=unit
```

Comment:
@candysmurf, is there an update on adding "integration" to the test matrix?  (see https://docs.travis-ci.com/user/docker/)